draft-specifications/Model.md

```diff
@@ -235,50 +284,69 @@ A struct with unique, unordered string keys and values of heterogeneous type.
 
 ## Tagged
 
-A value with a string tag.
+A [Value](#value) with a tag.
+The value may be any value.
+The tag is a string of Unicode code points excluding surrogates (U+D800-U+DFFF).
 
 > OCapN provides a small number of container types with minimal semantics.
 > Tags allow protocols to emerge from values annotated to indicate a richer
 > variety of semantics.
 > Concretely, a list might be differentiated from or hinted to be an ordered
 > list, sorted list, or a set of unique values.
 >
-> - **Guile**: currently a Guile record labeled `'(syrec ...)` (**imported**)
+> - **Guile**: currently a Guile record labeled `'(make-tagged 'tagName 10)` (**imported**)
 > - **JavaScript**: an object with the key `Symbol.for('passStyle')` and value
 >   `tagged`, with a string tagName and a single `"payload"` field carrying the tagged value.
 >   ```js
 >   {
 >     [Symbol.for('passStyle')]: 'tagged',
->     [Symbol.toStringTag]: tagName,
+>     [Symbol.toStringTag]: 'tagName',
 >     payload
 >   }
 >   ```
-> - **Python**: `Tagged(tag, value)` where `tag` is a `string` and `Tagged` is
+> - **Python**: `Tagged('tagName', value)` where `tag` is a `string` and `Tagged` is
 >   imported from `ocapn`.
 >
 > OCapN tagged data have a single value (are not variadic) [November 14, 2023](https://github.com/ocapn/ocapn/issues/94)
 >
 > What are tagged values for? https://github.com/ocapn/ocapn/issues/52
 
-## Remotable
+## Error
+
+A value capturing the reason for rejecting a delivery.
+
+> - **Guile**: to be proposed
+> - **JavaScript**: a JavaScript Error object
+> - **Python**: to be proposed
+>
+> We have not yet converged on any particular details about the modeling of
+> errors. The purpose of errors is typically to indicate that some requested
+> operation failed. The purpose of the contents of errors is to preserve and
+> convey diagnostic information, mostly to help debug problems, such as the
+> root cause of a surprising failure. This is a best-efforts obligation, for
+> which we have not yet decided either what contents are required, nor what is
+> allowed, nor what must be preserved as errors are passed from one site to
+> another. Until these details are decided, the only hard requirement is that
+> an error round trip to an error. We avoid any interpretation for now as to
+> whether it is the "same" error.
 
-A value that can receive messages.
+# Reference
```

Comment:
Saying `Reference` rather than `Capability` seemed good to me at first. But we name the protocol `ocapn` and make a big deal of it being a distributed capability protocol. This is where those capabilities appear. Calling these capabilities means there's one less concept to explain.

Reply:
Pending feedback from @cwebber and @tsyesika, I’ll for now use Reference as the primary name and Capability as its epithet in parentheses.